sys-apps/smcipmitool/smcipmitool-2.19.0.171201.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver java-vm-2
```

Comment:
Does this package need Java at all to actually work? I see no JDK/JRE specified in DEPEND/RDEPEND which makes me wonder why this eclass is inherited.

Reply:
Yes, it needs Java. I've added the missing dependency.
I am using the class for "${JAVA_VM_SYSTEM}" to create the symlink.